qiita_pet/handlers/api_proxy/prep_template.py

```diff
@@ -107,16 +107,33 @@ def prep_template_ajax_get_req(user_id, prep_id):
     name = "Prep information %d" % prep_id
     pt = PrepTemplate(prep_id)
 
-    job_id = r_client.get(PREP_TEMPLATE_KEY_FORMAT % prep_id)
-    if job_id:
-        redis_info = loads(r_client.get(job_id))
-        processing = redis_info['status_msg'] == 'Running'
-        if processing:
-            alert_type = 'info'
-            alert_msg = 'This prep template is currently being updated'
+    job_info = r_client.get(PREP_TEMPLATE_KEY_FORMAT % prep_id)
+    if job_info:
+        job_info = loads(job_info)
+        job_id = job_info['job_id']
+        if job_id:
+            redis_info = loads(r_client.get(job_id))
+            processing = redis_info['status_msg'] == 'Running'
```

Comment:
If that's the case, then it would be worth checking if the same is possible in `sample_template.py`.